yarn.lock

```diff
@@ -76,6 +76,18 @@
   version "0.0.39"
   resolved "https://registry.yarnpkg.com/@types/estree/-/estree-0.0.39.tgz#e177e699ee1b8c22d23174caaa7422644389509f"
 
+"@types/jest-diff@*":
```

Comment:
Please remove this change as it will conflict. 
We might remove lock files as this will be common problem for all contributors